clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1649,6 +1650,8 @@ check_schedule_file()
             gen_marker(TID_BASE + 2, TRACE_MARKER_TYPE_TIMESTAMP, TIMESTAMP_BASE + 3),
             gen_marker(TID_BASE + 2, TRACE_MARKER_TYPE_CPU_ID, CPU_BASE + 2),
             gen_instr(TID_BASE + 2, 4),
+            gen_marker(TID_BASE + 2, TRACE_MARKER_TYPE_TIMESTAMP, TIMESTAMP_BASE + 3),
```

Comment:
Add a comment explaining what this is testing.

Reply:
Done.